### PR TITLE
test case for verfying elements of reports page

### PR DESCRIPTION
**Pull request by iamjayesh02**

### CODE REVIEW

**Comment by prabhakarbattula** [comment]
@iamjayesh02 

Indentation is missing.  Please format properly.

File: aceinvoice/Testcases/reports_page/Verify_Elements_Reports_Page.java [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+        String weekly_timesheets = OpenBrowser.driver.findElement(By.xpath("//span[text()='Weekly Timesheets']")).getText();
+        assertEquals(weekly_timesheets,"Weekly Timesheets");
+        String weekly_timesheets_message = OpenBrowser.driver.findElement(By.xpath("//a[@href=\"/organizations/e87dec6b56b90810731b/reports/team_summary\"]/p")).getText();
+        assertEquals(weekly_timesheets_message,"See hours logged by team members for any given week.");
+
+        //Hover weekly timesheets
+        String weekly_timesheets_text_color = OpenBrowser.driver.findElement(By.xpath(xpath + "/li[1]/a")).getCssValue("color");
+        String weekly_timesheets_bk_color = OpenBrowser.driver.findElement(By.xpath(xpath + "/li[1]/a")).getCssValue("background-color");
+        assertNotEquals(weekly_timesheets_text_color,weekly_timesheets_bk_color);
+
```
Comment (prabhakarbattula) [inline aceinvoice/Testcases/reports_page/Verify_Elements_Reports_Page.java]: In line 51 or 52 or earlier, there is no invoking of hover action.

1) you need to hover on the element
2) obtain the background colour and assign it to a variable say `back-colour`
3) verify that the `back-colour`==#9e6aed;

**Comment by prabhakarbattula** [comment]
@iamjayesh02 

Please invoke hover and verify the background colour.

File: aceinvoice/Testcases/reports_page/Verify_Elements_Reports_Page.java [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+    @Test()
+    public void verify_reports_page()
+    {
+        //Verify reports
+        String reports = OpenBrowser.driver.findElement(By.xpath("//h1[text()='Reports']")).getText();
+        assertEquals(reports,"Reports");
+
+        //Hover weekly timesheets
+        //Get color  in form of rbga
+        String weekly_timesheets_color = OpenBrowser.driver.findElement(By.xpath(xpath + "/li[1]/a")).getCssValue("color");
```
Comment (prabhakarbattula) [inline aceinvoice/Testcases/reports_page/Verify_Elements_Reports_Page.java]: Should be background-color
.getCssValue('background-color')

**Comment by prabhakarbattula** [comment]
@iamjayesh02 

Hover action was not invoked.  
css property should be background-color instead of just color.

Please go through the stackoverflow link.
https://stackoverflow.com/questions/17293914/how-to-perform-mouseover-function-in-selenium-webdriver-using-java

File: aceinvoice/Testcases/reports_page/Verify_Elements_Reports_Page.java (hunk 2 above)
Comment (iamjayesh02) [reply aceinvoice/Testcases/reports_page/Verify_Elements_Reports_Page.java]: @prabhakarbattula  yes, but in this case .getCssValue('color') it was giving me background color. That's why I took color. But sure I will modify test case.

**Pull request merged**